app/helpers/blacklight_helper.rb

```diff
@@ -71,4 +71,18 @@ def render_table_of_contents_value
       table_contents[0]
     end
   end
+
+  def render_truncated_list(d_presenter, sinai_index)
+    if sinai_index.length.positive?
+      sinai_index.each do |_field_name, field|
+        trunc = "<div class='metadata-value-index--sinai'>"
+        value = (d_presenter.field_value field)
+        field_values = value.split("&nbsp;|&nbsp;")
+        trunc += field_values[0..2].join("&nbsp;|&nbsp;")
+        trunc += "&nbsp;|&nbsp;..." if field_values.length > 3
+        trunc += "</div>"
+        return trunc.html_safe
```

Comment:
I don't think we want to return from inside an .each loop - what is this method doing? Where is its unit test?